### PR TITLE
Update map user manual

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for issue #287

### CODE REVIEW

File: docs/src/user/manual/map.md [hunk 1]
```diff
@@ -7,9 +7,63 @@ The user is provided with a visual map to interact with to understand the extent
...
+    ![OpenStreetMap Pioneer View](./img/map-6.png)
+    
+    - Designed for exploration and outdoor activities.
+    - Highlights topographic details, terrain, and natural features, making it suitable for adventurous and nature-related mapping.
+
+4. **OpenStreetMap Neighbourhood View:**
+
+    ![OpenStreetMap Street View](./img/map-7.png)
+
+    - Emphasizes localized mapping for neighborhood-level details.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/map.md:68]:
```suggestion
    - Emphasizes localized mapping for neighbourhood-level details.
```

File: docs/src/user/manual/map.md [hunk 2]
```diff
@@ -7,9 +7,63 @@ The user is provided with a visual map to interact with to understand the extent
...
+    - Useful for focusing on smaller-scale features, such as streets, parks, and amenities within specific neighborhoods or community areas.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/map.md:69]:
```suggestion
    - Useful for focusing on smaller-scale features, such as streets, parks, and amenities within specific neighbourhoods or community areas.
```

File: docs/src/user/manual/map.md [hunk 3]
```diff
@@ -7,9 +7,63 @@ The user is provided with a visual map to interact with to understand the extent
...
+            - Keyword
+            - Name
+        
+        This detailed information provides insights into the metadata record, aiding users in understanding its specific attributes and content.
+
+2. **Zoom functionality:** This allows users to zoom in and out on the map.
+ 
+3. **Base map options:** This allows users to choose a specific base map.
+
+![base map options](./img/map-10.png)
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/map.md:41]: Just to help with formatting we can resize some images

```suggestion
![base map options](./img/map-10.png){: style="height:200px"}
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

File: docs/src/user/manual/map.md (hunk 3 above)
Comment (ketankartoza) [reply docs/src/user/manual/map.md:41]: okay

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Pull request merged**